### PR TITLE
Recreated the website as an Ember Octane app

### PR DESCRIPTION
**Pull request by ijlee2**
Hello. I removed all existing files and installed an Octane app (v3.15.1). I copied code from [my existing repo](https://github.com/ijlee2/ember-code-snippet-tutorial) (use Prism for syntax highlighting) and ran tests.

I updated `README.md` to reflect an Ember Octane app, and created `CONTRIBUTING.md` to keep readme concise. Please feel free to edit them as needed.

Let me know if you have questions!

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -11,7 +11,7 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 indent_style = space
-indent_size = 2
+indent_size = 4
```
Comment (rwjblue) [inline .editorconfig]: Can you reset this? `2` is what the default blueprint generates, and is much more common.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        'plugin:ember/recommended'
+    ],
+    env: {
+        browser: true
+    },
+    rules: {
+        'ember/no-jquery': 'error',
+
+        // AirBnB style
+        'array-bracket-spacing': [
```
Comment (rwjblue) [inline .eslintrc.js]: I'd prefer to keep this repo as close to the default blueprint as possible. Can you reset this file to the default setup?

File: .template-lintrc.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+    extends: 'octane',
+    rules: {
+        'block-indentation': 4
```
Comment (rwjblue) [inline .template-lintrc.js]: ✂️ 😸

File: LICENSE [hunk 4]
```diff
@@ -1,21 +0,0 @@
-MIT License
```
Comment (rwjblue) [inline LICENSE]: This file should be brought back.

File: README.md [hunk 5]
```diff
@@ -1,37 +1,38 @@
...
-Check out [the deployed app](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/)!
+Check out [the deployed app](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/). Yep, it uses Ember Octane!
 
-See side-by-side examples of Classic Ember vs [Octane Edition](https://emberjs.com/editions).
+You can see side-by-side examples of [Ember Octane](https://emberjs.com/editions/octane) and Ember Classic.
 
-This uses [Prism](https://prismjs.com/) code hightlighting with the [file highlight](https://prismjs.com/plugins/file-highlight/) plugin.
 
-Inspired by [YouMightNotNeedJQuery](http://youmightnotneedjquery.com/).
+## Running Locally
```
Comment (rwjblue) [inline README.md]: Should this also be in `CONTRIBUTING.md`? Seems like the only reason to run locally would be to contribute.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,63 @@
+{
+    "name": "ember-octane-vs-classic-cheat-sheet",
+    "version": "0.0.0",
+    "private": true,
+    "description": "Small description for ember-octane-vs-classic-cheat-sheet goes here",
```
Comment (rwjblue) [inline package.json]: Should fill this in

File: package.json [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+        "ember-intl": "^4.2.2",
+        "ember-load-initializers": "^2.1.1",
+        "ember-maybe-import-regenerator": "^0.1.6",
+        "ember-prism": "^0.5.0",
+        "ember-qunit": "^4.6.0",
+        "ember-resolver": "^7.0.0",
+        "ember-source": "~3.15.0",
+        "ember-test-selectors": "^3.0.0",
+        "eslint": "^6.1.0",
+        "eslint-config-airbnb-base": "^14.0.0",
```
Comment (rwjblue) [inline package.json]: 🔪

**Review by rwjblue — CHANGES_REQUESTED** [review]

**Comment by rwjblue** [comment]
It would also be great to see this running somewhere (maybe netlify or your own forks gh-pages), to poke at it a bit...

File: .editorconfig (hunk 1 above)
Comment (ijlee2) [reply .editorconfig]: Yep, will do this afternoon. Thanks!

File: README.md (hunk 5 above)
Comment (ijlee2) [reply README.md]: Yep, I think it may be a good idea. Will move the text to contributing.

File: LICENSE (hunk 4 above)
Comment (ijlee2) [reply LICENSE]: Good catch, thanks!

**Comment by ijlee2** [comment]
> It would also be great to see this running somewhere (maybe netlify or your own forks gh-pages), to poke at it a bit...

@rwjblue Yep, definitely. You can find a demo at https://ember-code-snippet-tutorial.herokuapp.com/ . Here's the current website: https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/ .

I've applied all of your suggestions, thanks for the help. Can you take a look again some time?

File: .template-lintrc.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+  extends: 'octane',
+  rules: {
+    'no-bare-strings': true
```
Comment (ijlee2) [inline .template-lintrc.js:6]: Without specifying the rule, the CLI displays a warning message.

Comment (rwjblue) [reply .template-lintrc.js:6]: Ahh, righto (this is because we have ember-intl).

**Review by rwjblue — APPROVED** [review]
Seems good to me!

I'll leave it to @jenweber / @mansona / @locks / et al to land...

**Comment by locks** [comment]
Why does the app use ember-intl?

**Comment by ijlee2** [comment]
> Why does the app use ember-intl?

I used it to collect subsection titles and descriptions into one file. I also used nested keys to indicate where titles and descriptions will be used and to create the entire site with one nested for-loop.

By storing titles and descriptions in one place, in an organized manner, I am hoping that new contributors would readily see how to make changes, so that the site will be updated and maintained regularly.

A secondary benefit is, using ember-intl can give contributors the opportunity to work on translating the site in their languages. While this may be hard to initiate and maintain with a large-scale project like the Ember Guides, I think it may be possible with the Octane vs. Classic Cheat Sheet.

**Comment by ijlee2** [comment]
I'll close this PR as it has been a few months since we considered reviewing this for merge. Ember 3.15 is no longer the latest and GitHub shows a couple of files in merge conflict.

Let's make a new PR if we still think it's a good idea to rewrite the site as an Ember app.

**Comment by jenweber** [comment]
Hiya, sorry for forgetting about this! I thought we had merged it and the remaining task was deploying. I would be in favor of making this the new `master` and then backporting any of the PRs that were made to master in the meantime. I'll DM you.

**Pull request closed without merging**